XenModel/Messages.resx

```diff
@@ -11496,6 +11572,14 @@ To learn more about the [XenServer] Performance Reporting capabilities feature o
 
 To learn more about the [XenServer] Role Based Access Control feature or to start a [XenServer] trial, click the button below.</value>
   </data>
+  <data name="UPSELL_BLURB_VMSS" xml:space="preserve">
+    <value>Upgrade your [XenServer] license to enable VM Snapshot Schedule. VM Snapshot Schedule allows you to create automatic snapshots for your critical VMs.</value>

```

Comment:
Ah! So there seems to be a disconnect in what we would want to call this feature after all. I recall, from our discussion, that we wanted to have this as `VM Snapshot Schedule` and looks like we want to different now.
I would then prefer to have `VM Scheduled Snapshot` be consistent, in the XC, XAPI, SDK, etc. Does this sound acceptable?